src/cap-description.adoc

```diff
@@ -385,16 +385,14 @@ definitions in xref:cap_encoding_ct[xrefstyle=short] and
 xref:cap_encoding_cb[xrefstyle=short].
 
 ```
-Ac = a[E + MW - 1:E + MW - 3]
-Bc = B[MW - 1:MW - 3]
-Tc = T[MW - 1:MW - 3]
-R  = Bc - 1
+A = a[E + MW - 1:E]
+R  = B - 2^MW-2^
 ```
 
 .Calculation of top address correction
 [#cap_encoding_ct,options=header,cols="^1,^1,^1",width="40%",align="center"]
 |==============================================================================
-| A~c~ < R | T~c~ < R | c~t~
+| A < R    | T < R    | c
```

Comment:
This one is still ct to define the variable above for top

Reply:
Quite right. Fixed